cfgov/v1/models/organisms.py

```diff
@@ -170,6 +171,7 @@ class ItemIntroduction(blocks.StructBlock):
     class Meta:
         icon = 'form'
         template = '_includes/organisms/item-introduction.html'
+        classname = 'block__flush-top'
```

Comment:
I don't think the 50/50 group is supposed to change. I'll try to double check this w/ the new info-unit changes that were merged.